vizro-core/src/vizro/models/_components/form/date_range_picker.py

```diff
@@ -0,0 +1,71 @@
+from typing import List, Literal, Optional
+
+import dash_mantine_components as dmc
+from dash import html
+
+try:
+    from pydantic.v1 import Field, validator
+except ImportError:  # pragma: no cov
+    from pydantic import Field, validator
+
+
+from vizro.models import VizroBaseModel
+from vizro.models._action._actions_chain import _action_validator_factory
+from vizro.models._components.form._form_utils import validate_date_picker_value, validate_max_date
+
+
+class DateRangePicker(VizroBaseModel):
```

Comment:
Same comments I had on the DatePicker apply here (docstrings, typing, arguments etc.) :)